src/Binder.php

```diff
@@ -61,12 +61,13 @@ public static function createSimpleProductionBinder()
     /**
      * @param mixed $request
      * @param mixed $object
+     * @param string[] $validationGroups Groups to apply validation to
      *
      * @return BindResult
      *
      * @throws \Exception
      */
-    public function bind($request, $object)
+    public function bind($request, $object, array $validationGroups = array())
```

Comment:
For some reason I thought those were only added in 5.6, never knew it was added all the way back in 5.4. Let me update the PR.